.github/workflows/platform.set-avm-github-pr-labels.yml

```diff
@@ -0,0 +1,42 @@
+# Workflow for adding labels to newly created PRs
+name: .Platform - Set AVM GitHub PR labels
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  SetAvmGitHubPrLabels:
+    if: github.repository == 'Azure/bicep-registry-modules'
+    name: Set-AvmGitHubPrLabels
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: tibdex/github-app-token@v2
```

Comment:
@rahalan since I'm not confident enough on how this external action treats secrets, and given this pipeline is expected to run from forks, so anyone can trigger it, I'd appreciate a second opinion on the workflow itself

Reply:
@ChrisSidebotham can you have a look? Thanks.

Reply:
@eriqua - Not sure what the concern is here, you would need access to the secrets to leverage the job & token. This is a standard external task which is used by CAE in other repo's too @jtracey93 can confirm

Reply:
yup you need access to the private key which is in a secret in this repo.

I thought this repo was only meant to run on this repo so if so add the if condition to ensure that

Reply:
@jtracey93 that check is already in the PR, see 
![image](https://github.com/user-attachments/assets/5b6f3ad0-2de2-42ed-88da-2173f6d3638f)

Reply:
Thanks @ChrisSidebotham and @jtracey93 for confirming and @rahalan for your patience! Approving